src/content/community/releases/_index.en.md

```diff
@@ -3,6 +3,35 @@ date = 2020-08-24T11:35:16+02:00
 title = "Releases"
 weight = 40
 +++
+<!-- markdownlint-disable no-duplicate-heading/no-duplicate-header -->
+## v0.13.0
+
+### New features
+
+* All Submariner container images are now available for x86-64 and ARM64 architectures.
+* Support was added in `subctl cloud prepare` to deploy Submariner on OpenShift on Microsoft Azure. This automatically configures the
+underlying Azure cloud infrastructure to meet Submariner's prerequisites.
+* Added a new, more robust, support for connecting clusters that use the OVNKubernetes CNI plugin in non-Globalnet deployments. Note that
```

Comment:
```suggestion
* Added support for connecting clusters that use the OVNKubernetes CNI plugin in non-Globalnet deployments. Note that
```

Is the feature new or improved? It doesn't seem like it can be both new and more robust. IMHO it would read more clearly to have one or neither adjective.

Reply:
Yes, see similar comment from Tom.